nih-sparc/sparc-app
# update to protocols

# Description

Change location from which protocol links are fetched in the dataset details page.
https://www.wrike.com/open.htm?id=738446289


## Type of change
# How Has This Been Tested?
Locally tested:
1) Check that datasets with references use the references to protocols
2) Check that datasets without references use the records
3) Check that datasets without references only show protocols with DOI

# Checklist:
